torch/_subclasses/meta_utils.py

```diff
@@ -1565,7 +1565,7 @@ def __call__(
 
         # Filter out cases we don't support
         # TODO: This can probably be simplified quite a bit
-        if isinstance(t, torch.Tensor) or is_traceable_wrapper_subclass(t):
+        if isinstance(t, torch.Tensor):
```

Comment:
The doc in `is_traceable_wrapper_subclass` says:
```
Returns whether or not a tensor subclass that implements __torch_dispatch__
    is 'traceable' with torch.compile.
```
which (to me) implies that it will always be a tensor subclass.